packages/plugin-packages/mashroom-portal/src/backend/utils/model_utils.ts

```diff
@@ -58,7 +58,7 @@ export const getPageData = async (sitePath: string, friendlyUrl: string | undefi
 
     let pageRef = null;
     if (friendlyUrl) {
-        pageRef = await portalService.findPageRefByFriendlyUrl(site, friendlyUrl);
+        pageRef = await portalService.findPageRefByFriendlyUrl(site, decodeURIComponent(friendlyUrl));
```

Comment:
Is this really the right place to do the decoding of the friendlyUrl? like this the same problem could occur in other places; 

this should be done in renderPortalPage()